recipe/meta.yaml

```diff
@@ -1,11 +1,11 @@
-{% set version = "2022.02.2" %}
+{% set version = "2022.02.2+485" %}
```

Comment:
```suggestion
{% set version = "2022.02.2+485" %}
{% set upstream_build_num = version.split("+")[1] %}
# reset the below to 0 when bumping upstream build num.
{% set cf_build_num = 1 %}
```

Reply:
Ensuring that this is bot-update-compatible is the whole point of this PR, so let's make sure this proposal achieves that before committing to it.

Reply:
Thanks, fixed